client/src/main/com/sinch/sdk/models/Configuration.java

```diff
@@ -20,14 +21,16 @@ private Configuration(
       String oauthUrl,
       String numbersUrl,
       SMSRegion smsRegion,
-      String smsUrl) {
+      String smsUrl,
+      String verificationUrl) {
```

Comment:
Here you are adding only the URL as new configuration parameter, but I think we need to add also the `application key` and `application secret` from the voice dashboard as the verification API works with these application keys and not the ones defined under a project Id (Access keys)